src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/Interop/SafeMsQuicBufferHandle.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.InteropServices;
+using static System.Net.Quic.Implementations.MsQuic.Internal.MsQuicNativeMethods;
+
+namespace System.Net.Quic.Implementations.MsQuic.Internal
+{
+    internal sealed unsafe class SafeMsQuicBufferHandle : SafeHandle
+    {
+        public int Count;
```

Comment:
Can this be renamed `Capacity` just so we don't confuse the fill count in the _state with this?

```suggestion
        public int Count { get; }
```